psi4/driver/qcdb/libmintsbasissetparser.py

```diff
@@ -311,6 +315,37 @@ def parse(self, symbol, dataset):
                                         gaussian_type, 0, center, 0, 'Unnormalized'))
                                     shell_list.append(ShellInfo(am2, contractions2, exponents,
                                         gaussian_type, 0, center, 0, 'Unnormalized'))
+
+                                elif len(shell_type) >= 3 and shell_type[:2] == 'L=':
```

Comment:
I'm wondering if it wouldn't be cleaner to combine `len(shell_type) == 1` and `len(shell_type) >=3 and shell_type[:2] == 'L='` cases, as the only line that differs is the am assignment. Then we don't have the same logic 3 times...

Also, is `L=am` a rigorous spec, or can it have spaces (eg. `L = 5`)? Because the latter won't be parsed properly (one could use `.startswith("L")` and then `.split("=")[1]` to get the `am`).

Reply:
Done. And yes that should be rigorous.